subsys/bluetooth/controller/Kconfig

```diff
@@ -584,25 +584,39 @@ config BT_CTLR_SDC_CIS_SUBEVENT_LENGTH_US
 config BT_CTLR_CHANNEL_SOUNDING
 	bool "Channel Sounding support [EXPERIMENTAL]"
 	select EXPERIMENTAL
+	help
+		Enable Channel Sounding support.
 
 config BT_CTLR_SDC_CS_COUNT
 	int "Number of concurrent connections supporting CS procedure [EXPERIMENTAL]"
 	default 1
+	range 1 BT_MAX_CONN
 	depends on BT_CTLR_CHANNEL_SOUNDING
 	help
-	  Set the number of concurrent connections that can support Channel Sounding procedure.
+	  Set the number of concurrent connections that can support Channel Sounding.
+		Needs to be equal to or less than BT_MAX_CONN
 
 config BT_CTLR_SDC_CS_MAX_ANTENNA_PATHS
 	int "Max number of Channel Sounding antenna paths supported by the controller [EXPERIMENTAL]"
 	default 1
 	range 1 4
 	depends on BT_CTLR_CHANNEL_SOUNDING
+	help
+		Set the max number of antenna paths the controller supports. This differs from BT_CTLR_SDC_CS_NUM_ANTENNAS
+		since you can run a setup where local controller only has one antenna, but peer controller supports two antennas.
+		In this configuration, set BT_CTLR_SDC_CS_NUM_ANTENNAS to 1 and BT_CTLR_SDC_CS_MAX_ANTENNA_PATHS to 2
+		(,since local controller can send to both peer antennas, creating a total of 2 antenna paths)
 
 config BT_CTLR_SDC_CS_NUM_ANTENNAS
 	int "Number of antennas used in Channel Sounding procedure supported by the controller [EXPERIMENTAL]"
 	default 1
 	range 1 BT_CTLR_SDC_CS_MAX_ANTENNA_PATHS
 	depends on BT_CTLR_CHANNEL_SOUNDING
+	help
+		Set the max number of antennas supported by local controller. This differs from BT_CTLR_SDC_CS_MAX_ANTENNA_PATHS
+		since you can run a setup where both local controller and peer controller has 2 antennas.
+		For maximum antennas, set BT_CTLR_SDC_CS_NUM_ANTENNAS to 2 and BT_CTLR_SDC_CS_MAX_ANTENNA_PATHS to 4
```

Comment:
this "Fox maximum antennas [...] use two antennas" part confuses me a bit. Won't you need to set the following to use a 4:1 configuration?

BT_CTLR_SDC_CS_NUM_ANTENNAS=4
BT_CTLR_SDC_CS_MAX_ANTENNA_PATHS=4